Correct old references that don't match to current

vanessalama09 opened the pull request:
The goal of this PR to remove old verbiage such as EnsembleFrame, StatsFrame, and MetadataFrame and match them to how these components are currently referenced.

slabasan commented:
@vanessalama09 Two requests: (1) please resolve conflicts and (2) change aggregate statistics to aggregated statistics.

vanessalama09 commented:
Merge Last!

slabasan commented:
@vanessalama09 ok this one is ready for changes!

slabasan committed:
update outdated verbiage throughout

ensembleframe --> performance data table
metadataframe --> metadata table
statsframe --> aggregated statistics table
